packages/charting/src/components/VerticalBarChart/VerticalBarChart.types.ts

```diff
@@ -0,0 +1,141 @@
+import { ITheme, IStyle } from '../../Styling';
+import { IStyleFunctionOrObject } from '../../Utilities';
+
+export interface IVerticalBarChart {
+
+}
+
+export interface IDataPoint {
+  /**
+   * Independent value of the data point, rendered along the x-axis.
+   * If x is a number, then each y-coordinate is plotted at its x-coordinate.
+   * If x is a string, then the data is evenly spaced along the x-axis.
+   */
+  x: number | string;
+
+  /**
+   * Dependent value of the data point, rendered along the y-axis.
+   */
+  y: number;
+}
+
+export interface IVerticalBarChartProps {
+  /**
+   * Data to render in the chart.
+   */
+  data?: IDataPoint[];
+
+  /**
+   * Width of the chart.
+   */
+  width?: number;
+
+  /**
+   * Height of the chart.
+   */
+  height?: number;
+
+  /**
+   * Width of each bar in the chart.
+   */
+  barWidth?: number;
+
+  /**
+   * Number of ticks on the y-axis.
+   */
+  yAxisTickCount?: number;
+
+  /**
+   * Colors from which to select the color of each bar.
+   */
+  colors?: string[];
+
+  /**
+   * Label to apply to the whole chart.
+   */
+  chartLabel?: string;
+
+  /**
+   * Additional CSS class(es) to apply to the VerticalBarChart.
+   */
+  className?: string;
+
+  /**
+   * Theme (provided through customization.)
+   */
+  theme?: ITheme;
+
+  /**
+   * Call to provide customized styling that will layer on top of the variant rules.
+   */
+  styles?: IStyleFunctionOrObject<IVerticalBarChartStyleProps, IVerticalBarChartStyles>;
+}
+
+export interface IVerticalBarChartStyleProps {
+  theme: ITheme;
+  className?: string;
+  width: number;
+  height: number;
```

Comment:
We should jsdoc all these (I looked at other *StyleProps and they were documented)

Reply:
can be done in a future pr, but agree.